### PR TITLE
Jenkins config for LLDAP

### PR DESCRIPTION
**Pull request by themartinslife**
Created tutorial and sample config for Jenkins for LLDAP

### CODE REVIEW

File: example_configs/jenkins.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Configuration for Jenkins
+
+## Jenkins base setup
+
+To setup LLDAP for Jenkins navigate to Dashboard/Manage Jenkins/Security. 
+
+*Note: Jenkins LDAP plugin have to be installed!</br>*
```
Comment (nitnelave) [inline example_configs/jenkins.md:7]: Nit: *has* to be installed

File: example_configs/jenkins.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+*Note: "dc=example,dc=com" is basic setup, but it has to match your lldap dc.*
```
Comment (nitnelave) [inline example_configs/jenkins.md:8]: Nit: is the *default* configuration, you should replace it with your base DN

File: example_configs/jenkins.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+1) Set **Security Realm** to **LDAP**
+2) Click Add Server
+3) Setup config fiels as stated below
```
Comment (nitnelave) [inline example_configs/jenkins.md:12]: Fields

File: example_configs/jenkins.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+#### Server
+*(This can be replaced by server ip/your domain etc.)*
+```
+ldap://example.com:3890
+```
+### Advanced Server Configuration Dropdown
+
+#### root DN
+Leave empty
```
Comment (nitnelave) [inline example_configs/jenkins.md:24]: Root DN should probably be "dc=example, dc=com"

File: example_configs/jenkins.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+```
+
+#### Email Address LDAP attribute
+```
+mail
+```
+
+### Tips & Tricks
+- Always use Test LDAP settings so you won't get locked out. It works without password.
+- If you want to setup your permissions, go to Authorization setting and select Matrix-based security. Add group/user (it has to exist in LLDAP) and you can set him permissions. Note that Overall Read forbids users to read jenkins and execute actions. Administer gives full rights.
```
Comment (nitnelave) [inline example_configs/jenkins.md:75]: Not: you can *grant* him permissions.

**Review by nitnelave — CHANGES_REQUESTED** [review]
Looks good! A few formatting nits. Can you add it to the list in the readme as well?

**Review by themartinslife — COMMENTED** [review]
Fixed according to the review

**Comment by themartinslife** [comment]
@nitnelave Done!

**Review by nitnelave — APPROVED** [review]
Thanks!

**Pull request merged**